### ISSUE TITLE
Migration failure from 3.6.5

### ISSUE THREAD

**Issue by mijewe** [putyourlightson/craft-blitz#249]
### Describe the bug
The migration from 3.6.5 to 3.6.6 or 3.6.7 fails for me.

```
Output:

Yii Migration Tool (based on Yii v2.0.36)

    > drop index blitz_elementcaches_cacheId_elementId_unq_idx on {{%blitz_elementcaches}} ...    > add primary key blitz_elementcaches_cacheId_elementId_pk on {{%blitz_elementcaches}} (cacheId,elementId) ... done (time: 0.087s)
    > drop foreign key blitz_elementexpirydates_elementId_fk from table {{%blitz_elementexpirydates}} ... done (time: 0.012s)
    > drop index blitz_elementexpirydates_elementId_unq_idx on {{%blitz_elementexpirydates}} ... done (time: 0.045s)
    > add primary key blitz_elementexpirydates_elementId_pk on {{%blitz_elementexpirydates}} (elementId) ... done (time: 0.022s)
    > add foreign key blitz_elementexpirydates_elementId_fk: {{%blitz_elementexpirydates}} (elementId) references {{%elements}} (id) ... done (time: 0.030s)
    > drop index blitz_elementquerycaches_cacheId_queryId_unq_idx on {{%blitz_elementquerycaches}} ...    > add primary key blitz_elementquerycaches_cacheId_queryId_pk on {{%blitz_elementquerycaches}} (cacheId,queryId) ...Exception: SQLSTATE[42000]: Syntax error or access violation: 1280 Incorrect index name 'blitz_elementquerycaches_cacheId_queryId_unq_idx'
The SQL being executed was: ALTER TABLE `blitz_elementquerycaches` ADD CONSTRAINT `blitz_elementquerycaches_cacheId_queryId_pk` PRIMARY KEY (`cacheId`, `queryId`) (/xxx/vendor/yiisoft/yii2/db/Schema.php:677)
#0 /xxx/vendor/yiisoft/yii2/db/Command.php(1298): yii\db\Schema->convertException(Object(PDOException), 'ALTER TABLE `bl...')
#1 /xxx/vendor/yiisoft/yii2/db/Command.php(1093): yii\db\Command->internalExecute('ALTER TABLE `bl...')
#2 /xxx/vendor/yiisoft/yii2/db/Migration.php(437): yii\db\Command->execute()
#3 /xxx/vendor/craftcms/cms/src/db/Migration.php(362): yii\db\Migration->addPrimaryKey('blitz_elementqu...', '{{%blitz_elemen...', Array)
#4 /xxx/vendor/putyourlightson/craft-blitz/src/migrations/m200721_120000_add_primary_keys.php(64): craft\db\Migration->addPrimaryKey('blitz_elementqu...', '{{%blitz_elemen...', Array)
#5 /xxx/vendor/craftcms/cms/src/db/Migration.php(52): putyourlightson\blitz\migrations\m200721_120000_add_primary_keys->safeUp()
#6 /xxx/vendor/craftcms/cms/src/db/MigrationManager.php(232): craft\db\Migration->up(true)
#7 /xxx/vendor/craftcms/cms/src/db/MigrationManager.php(148): craft\db\MigrationManager->migrateUp(Object(putyourlightson\blitz\migrations\m200721_120000_add_primary_keys))
#8 /xxx/vendor/craftcms/cms/src/services/Updates.php(227): craft\db\MigrationManager->up()
#9 /xxx/vendor/craftcms/cms/src/console/controllers/MigrateController.php(328): craft\services\Updates->runMigrations(Array)
#10 [internal function]: craft\console\controllers\MigrateController->actionAll()
#11 /xxx/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#12 /xxx/vendor/yiisoft/yii2/base/Controller.php(180): yii\base\InlineAction->runWithParams(Array)
#13 /xxx/vendor/yiisoft/yii2/console/Controller.php(181): yii\base\Controller->runAction('all', Array)
#14 /xxx/vendor/yiisoft/yii2/base/Module.php(528): yii\console\Controller->runAction('all', Array)
#15 /xxx/vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('migrate/all', Array)
#16 /xxx/vendor/craftcms/cms/src/console/Application.php(87): yii\console\Application->runAction('migrate/all', Array)
#17 /xxx/vendor/yiisoft/yii2/console/Application.php(147): craft\console\Application->runAction('migrate/all', Array)
#18 /xxx/vendor/yiisoft/yii2/base/Application.php(386): yii\console\Application->handleRequest(Object(craft\console\Request))
#19 /xxx/craft(22): yii\base\Application->run()
#20 {main}
```

As per [the comment in the other issue](https://github.com/putyourlightson/craft-blitz/issues/240#issuecomment-676144682) , running `SHOW INDEX FROM blitz_elementquerycaches;` returns 3 results:

```
Table	Non_unique	Key_name	Seq_in_index	Column_name	Collation	Cardinality	Sub_part	Packed	Null	Index_type	Comment	Index_comment
blitz_elementquerycaches	0	blitz_elementquerycaches_cacheId_queryId_unq_idx	1	cacheId	A	0	NULL	NULL		BTREE		
blitz_elementquerycaches	0	blitz_elementquerycaches_cacheId_queryId_unq_idx	2	queryId	A	0	NULL	NULL		BTREE		
blitz_elementquerycaches	1	blitz_elementquerycaches_queryId_fk	1	queryId	A	0	NULL	NULL		BTREE		
```

### To reproduce
Steps to reproduce the behaviour:
1. Run `./craft update blitz` from Blitz 3.6.5
2. Observe the error

### Versions
- Plugin version: 2.6.5
- Craft version: 3.5.5

**Comment by bencroker** [comment]
Thanks for the follow-up. Can you please try running this query?

```
DROP INDEX blitz_elementquerycaches_cacheId_queryId_unq_idx ON blitz_elementquerycaches;
```

This is run as part of the migration but unusually doesn't seem to do the trick in your environment.

**Comment by bencroker** [comment]
Released a further fix in version 3.6.8.
